index.html

```diff
@@ -16,14 +21,17 @@
     <div>
       <a id="menulogo" href="#">Mauricio Cantillo</a>
     </div>
-    <nav>
-      <img id="burgermenu" src="./img/burger.png" alt="menu icon">
-      <div id="menu">
+    <nav class="indexPriority">
+      <button class="burgermenu">
+        <i class="menuIcon material-icons">menu</i>
+        <i class="closeIcon material-icons">close</i>
+      </button>
+      <div class="menu">
```

Comment:
- [ ] Please make sure when the user clicks any of the mobile menu options(Portfolio, About...), the correct part of the page is displayed

- ![mn](https://user-images.githubusercontent.com/63972670/167738843-9d8b8212-8198-4536-87ef-ef30612953ab.PNG)